src/common/config/configmetadata.go

```diff
@@ -0,0 +1,92 @@
+package config
+
+import (
+	"sync"
+)
+
+// Constant for configure item
+const (
+	// Scope
+	UserScope   = "user"
+	SystemScope = "system"
+	// Group
+	LdapBasicGroup = "ldapbasic"
+	LdapGroupGroup = "ldapgroup"
+	EmailGroup     = "email"
+	UAAGroup       = "uaa"
+	DatabaseGroup  = "database"
+	BasicGroup     = "basic"
+	ClairGroup     = "clair"
+
+	// Type
+	IntType      = "int"
```

Comment:
how about create a type for the type of config item and set ValidateFunc as a func of this type?